src/classes/BDI_DataImportDeleteBTN_CTRL.cls

```diff
@@ -143,10 +152,20 @@ public with sharing class BDI_DataImportDeleteBTN_CTRL {
      * @return PageReference 
      */
     public PageReference close() {
-        String retUrl = ApexPages.currentPage().getParameters().get('retURL');
+        String uiTheme = UserInfo.getUiThemeDisplayed();
+        String url;
+        Schema.DescribeSObjectResult dataImportDescribe = UTIL_Describe.getObjectDescribe(
+                UTIL_Namespace.StrTokenNSPrefix(DATA_IMPORT_API_NAME));
```

Comment:
Instead of passing the object name as a string to the `StrTokenNSPrefix` method, let's use the version of `getObjectDescribe` that takes the SObjectType as its argument be used here.  Then the `DATA_IMPORT_API_NAME` final string could be removed above.

`Schema.DescribeSObjectResult dataImportDescribe = UTIL_Describe.getObjectDescribe(
                DataImport__c.SObjectType);`

Reply:
ah good call. fixed that